__tests__/endpoints.test.js

```diff
@@ -1,139 +1,183 @@
-                expect(article.votes).toEqual(expect.any(Number))
-                expect(article.article_img_url).toEqual(expect.any(String))
-                expect(article.comment_count).toEqual(expect.any(Number))
-            })
-        
-            expect(articles).toBeSortedBy('created_at', {descending: true})
-        })
-    })
-})
+        expect(articles).toBeSortedBy('created_at', { descending: true });
+      });
+  });
+});
 
 describe('GET /api/articles/:article_id', () => {
-    describe('200', () => {
-        test('200: responds with an article with the requested article_id', () => {
-            return request(app)
-            .get('/api/articles/1')
-            .expect(200)
-            .then(({body}) => {
-                const article = body.article
-                expect(Object.keys(article)).toEqual([
-                    'article_id',
-                    'title',
-                    'topic',
-                    'author',
-                    'body',
-                    'created_at',
-                    'votes',
-                    'article_img_url'
-                ])
+  describe('200', () => {
+    test('200: responds with an article with the requested article_id', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+          const article = body.article;
+          expect(Object.keys(article)).toEqual([
+            'article_id',
+            'title',
+            'topic',
+            'author',
+            'body',
+            'created_at',
+            'votes',
+            'article_img_url',
+          ]);
 
-                expect(article.article_id).toBe(1)
-                expect(article.title).toEqual(expect.any(String))
-                expect(article.topic).toEqual(expect.any(String))
-                expect(article.author).toEqual(expect.any(String))
-                expect(article.body).toEqual(expect.any(String))
-                expect(article.created_at).toEqual(expect.any(String))
-                expect(article.votes).toEqual(expect.any(Number))
-                expect(article.article_img_url).toEqual(expect.any(String))
-            })
-        })
-    })
-    describe('400', () => {
-        test('400: returs error for invalid :article_id', () => {
-            return request(app)
-            .get('/api/articles/forty-two')
-            .expect(400)
-            .then(({body}) => {
-                expect(body).toEqual({msg: 'invalid_text_representation'})
-            })
-        })
-    })
-    describe('404', () => {
-        test('404: returs error for valid but non-existent :article_id', () => {
-            return request(app)
-            .get('/api/articles/42')
-            .expect(404)
-            .then(({body}) => {
-                expect(body).toEqual({msg: 'ID not found'})
-            })
-        })
-    })
-})
+          expect(article.article_id).toBe(1);
```

Comment:
Remember we are requesting a specific article. We don't just want any data, but the data related to that article. Please change this test